admin/load.php

```diff
@@ -408,7 +408,7 @@ function perflab_render_pointer() {
 	?>
 	<script id="perflab-admin-pointer" type="text/javascript">
 		jQuery( function() {
-			// Pointer Options
+			// Pointer Options.
```

Comment:
This period is unnecessary, so let's remove.

Reply:
Hmm, interesting @mitogh! I guess I'd argue that this is not a "full sentence," but of course want to follow the coding standards.

Reply:
Yeah, it's correct that basically any doc comment needs to end with a period per WP doc standards.

Reply:
> Hmm, interesting @mitogh! I guess I'd argue that this is not a "full sentence," but of course want to follow the coding standards.

Right, this is mostly because the way the standard assumes what "full sentence" means as is just any complete line. @bethanylang